components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/mvkpert/yaml_outs/monthly_average_coarse.yaml

```diff
@@ -88,8 +88,8 @@ Fields:
     - isccp_cldtot
 
 output_control:
-  Frequency: 1
+  frequency: 1
   frequency_units: nmonths
-  MPI Ranks in Filename: false
+  mpi_ranks_in_filename: false
```

Comment:
This line should actually be removed. `MPI Ranks in Filename` is ONLY used by standalone tests, and it's NOT even set via yaml files (it's automatically set by the IO layer).

Let's remove it from all our testmods dirs, so we don't mislead users.

Reply:
Ok, done.